explorer/src/bin/index.rs

```diff
@@ -142,10 +154,10 @@ fn run_indexer(
     log::info!("Starting '{}'", id);
 
     indexer.begin(&mut pg)?;
-    let mut log_every = Interval::new(Duration::from_secs(1));
+    let mut one_second = Interval::new(Duration::from_secs(1));
```

Comment:
When we include crate types like this:
```
use std::{
    sync::{atomic::*, mpsc::*, Arc},
    thread::*,
    time::*,
};
```
Isn't it better to explicitly provide all the ones that are going to be used? For example the `Interval`

Reply:
It depends. For std/core, I don't mind glob including. But especially for unfamiliar 3rd party crates, I explicitly name all the imports I use. Unless there are a ton of imports, or only imports from a few crates. So it's a decision to make each time.

Reply:
Also, Interval is defined in this file, not a std::time thing.